modules/cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class.adoc

```diff
@@ -27,45 +65,56 @@ spec:
       type: RuntimeDefault
   containers:
   - name: qos-demo-ctr
-    image: <image-pull-spec>
+    image: quay.io/openshifttest/hello-openshift:openshift <1>
     resources:
       limits:
-        memory: "200Mi"
-        cpu: "1"
+        memory: "200Mi" <2>
+        cpu: "1" <3>
       requests:
-        memory: "200Mi"
-        cpu: "1"
+        memory: "200Mi" <4>
+        cpu: "1" <5>
     securityContext:
       allowPrivilegeEscalation: false
       capabilities:
         drop: [ALL]
 ----
+<1> This example uses a public `hello-openshift` image. Replace with the image that you want to use for your workload.
+<2> Sets the memory limit to 200MB.
+<3> Sets the CPU limit to one CPU.
+<4> Sets the memory request to 200MB.
+<5> Sets the CPU request to one CPU.
++
+[NOTE]
+====
+If you specify a memory limit for a container, but do not specify a memory request, {product-title} automatically assigns a memory request that matches the limit. Similarly, if you specify a CPU limit for a container, but do not specify a CPU request, {product-title} automatically assigns a CPU request that matches the limit.
+====
+--
 
-. Create the pod:
+.. Create the `Pod` resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f qos-example.yaml --namespace=qos-example
+----
 +
+.Example output
 [source,terminal]
 ----
-$ oc  apply -f qos-pod.yaml --namespace=qos-example
+pod/qos-demo created
 ----
 
-. View detailed information about the pod:
+.Verification
+
+* View the `qosClass` value for the pod by running the following command:
 +
 [source,terminal]
 ----
-$ oc get pod qos-demo --namespace=qos-example --output=yaml
+$ oc get pod qos-demo --namespace=qos-example --output=yaml | grep qosClass
 ----
 +
 .Example output
 [source,yaml]
 ----
-spec:
-  containers:
-    ...
-status:
-  qosClass: Guaranteed
+    qosClass: Guaranteed
 ----
-+
-[NOTE]
-====
-If you specify a memory limit for a container, but do not specify a memory request, {product-title} automatically assigns a memory request that matches the limit. Similarly, if you specify a CPU limit for a container, but do not specify a CPU request, {product-title} automatically assigns a CPU request that matches the limit.
```

Comment:
This node is useful. It prevents human mistake, both cpu and memory support this behavior - when only resources.limits are specified, the requests are assumed to have the same value = Guaranteed QoS happens.

Reply:
The note was just moved actually, it's not deleted. It's still there just after the new sample YAML.

Reply:
Ack, I see it now.